integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -15,8 +15,9 @@
  */
 package com.okta.sdk.tests.it
 
-
 import com.okta.sdk.client.Client
+import com.okta.sdk.resource.CSR
```

Comment:
CSR isn't a great class name,  Is this from the spec? Ideally that gets spelled out so the name has a bit more meaning.